File: abc/abc279/a.cpp
// https://atcoder.jp/contests/abc279/submissions/36792933

#include <bits/stdc++.h>
using namespace std;

int main() {
  cin.tie(0);
  ios_base::sync_with_stdio(0);

  int cnt = 0;
  char c;
  while(cin >> c) {
    if(c == 'v') cnt++;
    else cnt += 2;
  }

  cout << cnt;
  return 0;
}

File: abc/abc279/a.cs
// https://atcoder.jp/contests/abc279/submissions/46443552

using System;

class A
{
  static void Main()
  {
    string s = Console.ReadLine();
    int ans = 0;

    for(int i = 0; i < s.Length; i++)
    {
      if(s[i] == 'v')
      {
        ans++;
      }
      else
      {
        ans += 2;
      }
    }

    Console.WriteLine(ans);
  }
}

